projects/xerces-c/project.yaml

```diff
@@ -0,0 +1,5 @@
+homepage: "https://xerces.apache.org/"
```

Comment:
Does anyone from upstream approve this project? Can you ask someone and get their email please?